robosuite/utils/input_utils.py

```diff
@@ -245,6 +245,8 @@ def input2action(device, robot, active_arm="right", env_configuration=None):
         # Scale rotation for teleoperation (tuned for OSC) -- gains tuned for each device
         drotation = drotation * 50 if isinstance(device, SpaceMouse) else drotation * 1.5
         dpos = dpos * 125 if isinstance(device, SpaceMouse) else dpos * 75
+    elif controller.name== "OSC_POSITION":
```

Comment:
Please follow the coding convention and add an extra space before ==.

Reply:
Updated.